<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="chartLayout">
     <item>
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>341</width>
      <height>92</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="text">
        <string>Цвет фона</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="comboBoxClr">
       <item>
        <property name="text">
         <string>Красный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Зеленый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Синий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Черный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Белый</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="text">
        <string>Цвет линии</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxFon">
       <item>
        <property name="text">
         <string>Красный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Зеленый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Синий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Черный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Белый</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="lineEditAlg_2">
       <property name="text">
        <string>Алгоритм</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxAlgos">
       <item>
        <property name="text">
         <string>Парам. вид</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Канон. вид</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Брезенхем</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Средняя точка</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Библ. функция</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonClrear">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>480</y>
      <width>341</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGraphs">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>510</y>
      <width>341</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Провести замеры</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>210</y>
      <width>341</width>
      <height>93</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>Радиус окружности</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxR"/>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxB"/>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="text">
        <string>Полуоси а и b</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxA"/>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButtonCircle">
       <property name="text">
        <string>Построить окружность</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QPushButton" name="pushButtonEllipse">
       <property name="text">
        <string>Построить эллипс</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>110</y>
      <width>341</width>
      <height>93</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0" colspan="2">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="text">
        <string>Координаты центра</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="text">
        <string>Yc</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="text">
        <string>Xc</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxXc"/>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxYc"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>310</y>
      <width>344</width>
      <height>159</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="text">
        <string>Нач. Радиус окружности</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxAStart"/>
     </item>
     <item row="1" column="0">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxRStart"/>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="text">
        <string>Шаг изменения</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QSpinBox" name="spinBoxStepSpec"/>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButtonCircleSpec">
       <property name="text">
        <string>От-вать спектр окружности</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="text">
        <string>Нач. Полуоси а и b</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxBStart"/>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QPushButton" name="pushButtonEllipseSpec">
       <property name="text">
        <string>От-вать спектр эллипса</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="text">
        <string>Кол-во отрисовок</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QSpinBox" name="spinBoxNumsSpec"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
